backend/Pipfile

```diff
@@ -14,4 +14,4 @@ email-validator = "1.1.1"
 pyjwt = "2.1.0"
 
 [requires]
-python_version = "3.8.10"
+python_version = "3.9.7"
```

Comment:
This pipfile is ment for the server, do not push local changes :+1:

Reply:
Tard, æ spør jo kordan man fjerne de